src/System Application/App/Email/src/Email/Inbox/EmailRetrievalFilters.Table.al

```diff
@@ -0,0 +1,63 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+namespace System.Email;
+
+/// <summary>Holds information about the filters for retrieving emails.</summary>
+table 8885 "Email Retrieval Filters"
+{
+    Access = Public;
+    TableType = Temporary;
+
+    fields
+    {
+        field(1; Id; Integer)
+        {
+            DataClassification = SystemMetadata;
```

Comment:
What about setting DataClassification only once instead, in the table properties?